test/ScopeLinter/unused.coffee

```diff
@@ -273,3 +273,16 @@ describe "ScopeLinter/unused", ->
         ), {
             unused_variables: true
         }).should.have.length(0)
+
+
+    it "matches composed string field objects", ->
```

Comment:
This spec is failing. Should travis.yml also be added to show that this spec fails publicly?